src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work.");

```

Comment:
The message should indicate what type of `Machine` is working right now `Truck`, `Bulldozer` or `Excavator`. Consider making the message more informative to reflect that this is a Truck specifically.